README.md

```diff
@@ -1,9 +1,8 @@
-SilverStripe CronTask
-==========================
-
 [![CI](https://github.com/silverstripe/silverstripe-crontask/actions/workflows/ci.yml/badge.svg)](https://github.com/silverstripe/silverstripe-crontask/actions/workflows/ci.yml)
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/silverstripe/silverstripe-crontask/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/silverstripe/silverstripe-crontask/?branch=master)
-[![codecov](https://codecov.io/gh/silverstripe/silverstripe-crontask/branch/master/graph/badge.svg)](https://codecov.io/gh/silverstripe/silverstripe-crontask)
+[![Silverstripe supported module](https://img.shields.io/badge/silverstripe-supported-0071C4.svg)](https://www.silverstripe.org/software/addons/silverstripe-commercially-supported-module-list/)
+
+Silverstripe CronTask
```

Comment:
Looks like this is mean to be the h1 header - put this back at the top, slap a `#` before it, and remove the `====` line.

Reply:
Updated